x/erc20/types/utils.go

```diff
@@ -81,3 +83,30 @@ func EqualStringSlice(aliasesA, aliasesB []string) bool {
 
 	return true
 }
+
+func GenRandomCoinMetadata(r *rand.Rand) banktypes.Metadata {
+	randDescription := simtypes.RandStringOfLength(r, 10)
+	randTokenBase := "a" + simtypes.RandStringOfLength(r, 4)
+	randSymbol := strings.ToUpper(simtypes.RandStringOfLength(r, 4))
+
+	validMetadata := banktypes.Metadata{
+		Description: randDescription,
+		Base:        randTokenBase,
+		// NOTE: Denom units MUST be increasing
```

Comment:
@poorphd 
What does 'increasing' mean? 
Could you share some example?

Reply:
Denom units "MUST be increasing" means that the Exponent values in the DenomUnits list should be in ascending order. This is crucial for representing different denominations of the token, ensuring smaller units are listed before larger units. For example:

- Denom: "abase", Exponent: 0
- Denom: "base", Exponent: 18